### ISSUE TITLE
Bindings break a compilation

### ISSUE THREAD

**Issue by robertsawko** [robertsawko/OpenPBE#6]
The compilation of kernels break with OF3.0.1 on Arnaud's computer. Commenting out the bindings from `CMakeLists.txt` seems to fix it. 

Here's the error.

```
In file included from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/UILList.H:39:0,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/ILList.H:39,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/IDLList.H:35,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/entry.H:45,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/dictionary.H:53,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/applications/solvers/multiphase/twoPhaseEulerFoam/twoPhaseSystem/lnInclude/phaseModel.H:35,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/../breakupKernels/breakupKernel.H:39,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/../breakupKernels/CoulaloglouTavlarides.H:39,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/kernels.cpp:2:
/home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/label.H:61:1: error: ‘INT32_MIN’ was not declared in this scope
/home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/label.H:62:1: error: ‘INT32_MAX’ was not declared in this scope
In file included from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/UILList.H:40:0,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/ILList.H:39,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/IDLList.H:35,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/entry.H:45,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/dictionary.H:53,
                 from /home/arnaud/OpenFOAM/OpenFOAM-3.0.1/applications/solvers/multiphase/twoPhaseEulerFoam/twoPhaseSystem/lnInclude/phaseModel.H:35,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/../breakupKernels/breakupKernel.H:39,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/../breakupKernels/CoulaloglouTavlarides.H:39,
                 from /home/arnaud/projects/Coupling_PBM_ICING/OpenPBE/src/PBESystem/bindings/kernels.cpp:2:
/home/arnaud/OpenFOAM/OpenFOAM-3.0.1/src/OpenFOAM/lnInclude/uLabel.H:62:1: error: ‘UINT32_MAX’ was not declared in this scope
make[2]: *** [src/PBESystem/bindings/CMakeFiles/kernels.dir/kernels.cpp.o] Error 1
make[1]: *** [src/PBESystem/bindings/CMakeFiles/kernels.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by Kojirion** [comment]
Will make bindings optional - set _not_ to build by default.

**Comment by robertsawko** [comment]
Good idea, please add to the pull request.

**Comment by robertsawko** [comment]
Solved by #4
